students/18th/team2/kimgyudong/user/views.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+from django.views import View
+from django.http  import JsonResponse
+
+from .models import User
+
+class UserSignUp(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        
+        if "email" not in data or "password" not in data:
+            return JsonResponse({"message":"KEY_ERROR"}, status = 400)
+        
+        email    = data["email"]
+        password = data["password"]
+
+        if '@' not in email or '.' not in email:
```

Comment:
이 부분은 @ 과 . 을 한번에 처리해주실 수 있습니다. 아래와 같이요!
```suggestion
        if ('@' or '.') not in email:
```

Reply:
넵 논리 연산자 수정으로 커밋했습니다!

Reply:
재수정 후 논리연산자 수정으로 커밋했습니다!